docs/server/source/production-deployment-template/node-config-map-and-secrets.rst

```diff
@@ -123,6 +123,13 @@ receive requests from NGINX instance.
 It is set to ``80`` by default.
 
 
+vars.bigchaindb-wsserver-advertised-scheme
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``bigchaindb-wsserver-advertised-scheme`` is the protocol used to access
+the WebSocket API in BigchainDB. This can be set to 'wss' (default) or 'ws'.
```

Comment:
:+1: 
Better yet, I will rephrase that as per the existing documentation of default values.